app/src/main/java/de/westnordost/streetcomplete/about/AboutFragment.java

```diff
@@ -95,7 +96,7 @@ public boolean onPreferenceClick(Preference preference)
 			}
 		});
 
-		getPreferenceScreen().findPreference("email_feedback").setOnPreferenceClickListener(
+		getPreferenceScreen().findPreference(Prefs.EMAIL_FEEDBACK).setOnPreferenceClickListener(
 				new Preference.OnPreferenceClickListener()
 		{
```

Comment:
These are no real preferences, these keys are not persisted. I created the About fragment this way just to have a cheap list-of-items. Please revert.